integration-tests/features/stack_analysis_npm_direct_and_transitive.feature

```diff
@@ -129,7 +129,7 @@ Feature: Thorough stack analysis v3 API tests for Node ecosystem
 
     # SLA/SLO-related checks
     When I look at the stack analysis duration
-    Then I should see that the duration is less than 40 second
```

Comment:
This is a wrong test case. This is not the one which is failing. Secondly, this is going to be an assert sort of statement and not something to be used for the computation time. Go to the respective test case and make changes there